codespell_lib/data/dictionary.txt

```diff
@@ -16051,6 +16247,7 @@ uptim->uptime
 uptions->options
 upto->up to
 uptream->upstream
+uris->urls
```

Comment:
URI is as good as URL
https://en.wikipedia.org/wiki/Uniform_Resource_Identifier

Reply:
removed